cmd/clusterctl/api/v1alpha3/groupversion_info.go

```diff
@@ -26,7 +26,7 @@ import (
 
 var (
 	// GroupVersion is group version used to register these objects
-	GroupVersion = schema.GroupVersion{Group: "clusterctl.cluster.x-k8s.io", Version: "v1alpha3"}
+	GroupVersion = schema.GroupVersion{Group: "clusterctl.inventory.x-k8s.io", Version: "v1alpha3"}
```

Comment:
Are we introducing a different API group?

Reply:
I think that `clusterctl.inventory.x-k8s.io` makes more sense that `clusterctl.cluster` here.

And probably I will introduce a new api group for https://github.com/kubernetes-sigs/cluster-api/issues/1373#issuecomment-566252107, because the inventory lives in the cluster (it is a CRD), while the release series (or whatever name we will chose) are a component config

Reply:
I'd try to find a single common API group for all of these resources, wdyt?

Reply:
I will find weird to get some resources of the same API group living in the cluster and some others in a file on the provider repository, but in the end, I'm fine with whatever you suggest

Reply:
What about something like `tools.cluster.x-k8s.io` or `cli.cluster.x-k8s.io`, basically something that captures everything outside of the common known tools, but still under cluster.

Reply:
I have done a rollback on this change. Let's figure it out later when the new type comes into the game